client/src/util/helpers.ts

```diff
@@ -109,6 +115,11 @@ export default {
     return null;
   },
 
+  formatPrice(amount:number, locale?: SupportedLocale): string {
+    const formatPrice = new Intl.NumberFormat(locale);
```

Comment:
One quick thing: make sure you account for the case where locale is not defined!